wgpu/src/triangle.rs

```diff
@@ -137,6 +145,7 @@ impl Pipeline {
                     depth_bias: 0,
```

Comment:
can omit this

Reply:
Done in b689778ed94b34706bf97e5994a721a8648386a5.